TRB_label_assist/models.py

```diff
@@ -121,13 +121,13 @@ class Cluster(SetupMixin, metaclass=ABCMeta):
     """ blueprint for clustering models. """
 
     @abstractmethod
-    def fit(self, train_df,ct_entry=None):
+    def fit(self, train_df,unused=None):
         """ Fit the clustering algorithm.  
         
             Args: 
                 train_df (DataFrame): dataframe of labeled trips
-                ct_entry (List) : A list of Entry type of labeled and unlabeled trips 
-            
+                unused (List) : A list of Entry type of labeled and unlabeled trips which is not used in current function. 
```

Comment:
why is this `unused` in the abstract class? If it is never used, in any subclass, it can just be removed completely. But it is used, so it should stay and the comments should be updated. Same message throughout the changes in this file.